CET release notes - et/agent/5.5.0 (#2464)
harness/developer-hub: release-notes/continuous-error-tracking.md

<details>
<summary>2023 releases</summary>

#### June 27, 2023, versions ET-Service 5.24.3 and ET-Collector 5.24.3

##### What's new

This release does not include new features and enhancements.

##### Early access

This release does not include any early access features.

##### Fixed issues

> - Unable to edit notifications created in the SRM module when using the CET module.(CET-1295)

Reviewer: ```suggestion
- Notifications created in the SRM module could not be edited when using the CET module.(CET-1295)
```

Reviewer: Though this is passive voice, recommend making this a complete sentence based on Deepak's recent post.
